### PR TITLE
Rewriting contributing guidelines

### PR DESCRIPTION
**Pull request by zeroset**
The Contributing guidelines are outdated and need to be rewritten. This PR makes first step for new guidelines.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,58 +4,8 @@ Contributing to JuPedSim
 This project is mainly developed by a small group of researchers and students from [Jülich Research Center](http://www.fz-juelich.de/en) and [BUW](http://www.uni-wuppertal.de/).
```
Comment (chraibi) [inline CONTRIBUTING.md:4]: Please use these links instead
BUW: https://www.asim.uni-wuppertal.de/en.html
FZJ: https://www.fz-juelich.de/ias/ias-7/EN/Home/home_node.html

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -4,58 +4,8 @@ Contributing to JuPedSim
...
-* Download all dependencies, check if cmake tells you something is missing
-    * Boost >= 1.56
-    * Qt
-    * Vtk
-    
-* If everything compiled for the first time, you are free to start
-
-### Workflow
+## Workflow
 The branches **master** and **develop** are **protected**. You can **only push to your feature-branch.**
```
Comment (chraibi) [inline CONTRIBUTING.md:9]: develop branch? I think we don't use develop..

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -64,33 +14,12 @@ After doing that open a merge/pull request.
...
-- Board usage
-- Gitlab in general: features
-- Wiki vs Web
-#######################
-
-### FAQ
-
 ## Reporting bugs and requesting features
 If you got a question or a problem and need support from our team feel free to contact us.
 You can do this via [email](mailto:dev@jupedsim.org).
```
Comment (chraibi) [inline CONTRIBUTING.md:19]: I would remove this mail and keep things in GitHub's issue tracker.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -4,58 +4,8 @@ Contributing to JuPedSim
...
-    * Qt
-    * Vtk
-    
-* If everything compiled for the first time, you are free to start
-
-### Workflow
+## Workflow
 The branches **master** and **develop** are **protected**. You can **only push to your feature-branch.**
 Whenever you want to add a feature or fix an issue create a new feature-branch and only work on this branch.
 If you change what you wanted to merge the develop-branch into your feature-branch.
```
Comment (schroedtert) [inline CONTRIBUTING.md:11]: Should we explicitly mention, that contributors need to fork the repo beforehand?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -361,87 +133,3 @@ Comments have to be written in **English** everywhere. Please use markdown where
 
 ### Documenting new features
 Please update the changelog with every feature/fix you provide so we can keep track of changes for new versions of JuPedSim.
```
Comment (schroedtert) [inline CONTRIBUTING.md:135]: Do we keep the changelog up-to-date?

Comment (zeroset) [reply CONTRIBUTING.md:135]: I guess we are supposed to but didn't do recently

File: CONTRIBUTING.md (hunk 1 above)
Comment (zeroset) [reply CONTRIBUTING.md:4]: Changed

File: CONTRIBUTING.md (hunk 2 above)
Comment (zeroset) [reply CONTRIBUTING.md:9]: removed develop, but i think i have to rewrite the whole paragraph.

File: CONTRIBUTING.md (hunk 3 above)
Comment (zeroset) [reply CONTRIBUTING.md:19]: changed.

File: CONTRIBUTING.md (hunk 4 above)
Comment (zeroset) [reply CONTRIBUTING.md:11]: Maybe we should just refer to a github guide ...

**Comment by zeroset** [comment]
jpscore will be reimplemented in [simulator](https://github.com/JuPedSim/simulator) repository.

**Pull request closed without merging**